compiler/tools/fpp-to-cpp/test/port/ReturnTypePortAc.ref.cpp

```diff
@@ -0,0 +1,116 @@
+// ======================================================================
+// \title  ReturnTypePortAc.cpp
+// \author Generated by fpp-to-cpp
+// \brief  cpp file for ReturnType port
+// ======================================================================
+
+#include "Fw/Types/Assert.hpp"
+#include "Fw/Types/StringUtils.hpp"
+#include "ReturnTypePortAc.hpp"
+
+namespace M {
+
+  // ----------------------------------------------------------------------
+  // Input Port Member functions
+  // ----------------------------------------------------------------------
+
+  InputReturnTypePort ::
+    InputReturnTypePort() :
+      Fw::InputPortBase(),
+      m_func(nullptr)
+  {
+
+  }
+
+  void InputReturnTypePort ::
+    init()
+  {
+    Fw::InputPortBase::init();
+  }
+
+  void InputReturnTypePort ::
+    addCallComp(
+        Fw::PassiveComponentBase* callComp,
+        CompFuncPtr funcPtr
+    )
+  {
+    FW_ASSERT(callComp);
+    FW_ASSERT(funcPtr);
+
+    this->m_comp = callComp;
+    this->m_func = funcPtr;
+    this->m_connObj = callComp;
+  }
+
+  U32 InputReturnTypePort ::
+    invoke(U32 u)
+  {
+#if FW_PORT_TRACING == 1
+    this->trace();
+#endif
+
+    FW_ASSERT(this->m_comp);
+    FW_ASSERT(this->m_func);
+
+    return this->m_func(this->m_comp, this->m_portNum, u);
+  }
+
+#if FW_PORT_SERIALIZATION == 1
+
+  Fw::SerializeStatus InputReturnTypePort ::
+    invokeSerial(Fw::SerializeBufferBase& _buffer)
+  {
+    // For ports with a return type, invokeSerial is not used
```

Comment:
clang is giving me an unused parameter warning when compiling this code. F Prime currently turns off these warnings, but I think we should make our generated code comply with them going forward. Let's add the following line to the function body to suppress the warning:
```
(void) _buffer;
```

Reply:
Fixed!